readme.txt

```diff
@@ -106,6 +106,7 @@ Please note that our support for the checkout block is still experimental and th
 * Fix - Incorrect prices in Payment Request Button for certain currencies.
 * Fix - Updates to fraud protection.
 * Fix - Prevent Payment Request buttons from showing up in Composite Product pages. 
+* Add - Empty state flows on the admin WooCommerce > Subscriptions screen for stores with no subscriptions yet.
```

Comment:
@james-allan small discrepancy between the wording here and what's in the changelog.txt, just checking if this is intentional?

Reply:
Ah thanks for catching that. That wasn't intentional. I've reworded the changelogs to be a little more clear.